app/libs/README.txt

```diff
@@ -1 +1 @@
-TODO(jontayler): eliminate these in favor of those from a maven repo.
\ No newline at end of file
+TODO(johntaylor): eliminate these in favor of those from a maven repo.
```

Comment:
Actually, can you leave this as-is? The misspelling is deliberate (when you're called John Taylor you have to get creative with spellings.... https://en.wikipedia.org/wiki/John_Taylor_(bass_guitarist)

Reply:
lol how are people actually mistaking you for the founder of Duran Duran?